opencsp/common/lib/deflectometry/ImageProjectionSetupGUI.py

```diff
@@ -376,7 +376,7 @@ def save_to_hdf(self, file: str):
         self.get_user_data()
 
         # Save as HDF file
-        ImageProjection.save_to_hdf(self.display_data, file)
+        ImageProjection.save_params_to_hdf(self.display_data, file)

```

Comment:
I like these changes!
YFI it is still on my list to change all `save_as_hdf` calls to `to_hdf`... just haven't gotten around to it yet.